bodhi/server/notifications.py

```diff
@@ -38,6 +40,9 @@ def init(active=None, cert_prefix=None):
     if cert_prefix is not None:
         fedmsg_config['cert_prefix'] = cert_prefix
 
+    hostname = socket.gethostname().split('.', 1)[0]
+    fedmsg_config['name'] = 'bodhi.%s' % hostname
```

Comment:
This always overrides ```'name'``` from line 39 above. Is that intentional? If so, we should probably drop line 39.